docs/LazoApp.md

```diff
@@ -0,0 +1,554 @@
+`LazoApp` class provides a common set of apis that can be used at both client and server side. Your application has to extend `LazoApp` in `application.js` file. This is where you load environment specific configuration, set application level properties, and define application level CSS and JavaScript. Please check the Lazo Application Strucure for more information [LazoApplicationStrucutre](LazoApplicationStrucutre.md)
```

Comment:
Please remove "define application level CSS and JavaScript" or add a section that covers `app.json`, https://github.com/lazojs/lazo/wiki/Configuration#appjson